src/lib/core/WeaveMessageLayer.h

```diff
@@ -935,6 +936,8 @@ inline void WeaveMessageSourceToStr(char *buf, uint32_t bufSize, const WeaveMess
     WeaveMessageLayer::GetPeerDescription(buf, (size_t)bufSize, msgInfo);
 }
 
+extern WEAVE_ERROR GenerateWeaveDeviceId(uint64_t & deviceId);
```

Comment:
Can we rename this to GenerateWeaveNodeId()?  I will want to use this to generate random node ids for Weave client applications like the Device Manager running inside mobile application.

Reply:
done